cmd/nitro/nitro.go

```diff
@@ -519,18 +518,18 @@ func ParseNode(ctx context.Context, args []string) (*NodeConfig, *genericconf.Wa
 	return &nodeConfig, &l1Wallet, &l2DevWallet, l1Client, l1ChainId, nil
 }
 
-func applyNitroDevNetRollupParameters(k *koanf.Koanf) error {
```

Comment:
I think we probably also want to keep the devnet support for now if it's easy enough to do

Reply:
This version of nitro doesn't support the current devnet. The reinit will have new addresses, but I can add in a special error for the devnet or put a stub if it'd help clarify things.

Reply:
Ah right, nevermind, no need I think